common/values.yaml

```diff
@@ -11,10 +11,10 @@ deploymentContainer:
   port: 8080
   livenessProbe:
     path: /
-    port: 8080
+    port: ""
```

Comment:
I think this works to set it to an empty value, and should default to container port